.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Tell about typedoc's bug
+---
+<!--
+    Thank you for contributing to Typedoc! 
+    
+    First, please use a clear and descriptive title for the issue to identify the bug.
+
+    Please be as specific as possible! Provide codes, sreenshots, repositories, explanations, issues, links ... anything that could describe your point.
+-->
+
+- [ ] I have checked [issues with bug label](https://github.com/TypeStrong/typedoc/labels/bug) and found no duplicates
+
+## Expected Behavior
+<!--
+    According to the code or documentation, how is Typedoc supposed to work?
+-->
+
+## Actual Behavior
+<!--
+    What does Typedoc fail to do?  
+-->
+
+## Steps to reproduce the bug
+<!--
+    Tell us how we could reproduce the bug. 
```

Comment:
```suggestion
    Tell us how we could reproduce the bug. Possible ways are providing:
    - Typedoc command (typedoc --out ...)
    - Configuration (typedoc.js or your custom option file)
    - Code that causes a bug in the documentation generated
    - Project (Github repository, CodePen, ...) where the issue can be reproduced.
```
Is this alright? I think there are a few more examples of how an issue can be reproduced, but I'm just afraid it's going to be too long.

Reply:
Sorry, didn't see this comment earlier. What you ended up with looks good.